readme.md

```diff
@@ -660,15 +660,15 @@ Use <kbd>up</kbd>/<kbd>down</kbd> to navigate. Use <kbd>tab</kbd> to cycle the l
 | warn | `string` | Message to display when selecting a disabled option |
 | choices | `Array` | Array of strings or choices objects `[{ title, description, value, disabled }, ...]`. The choice's index in the array will be used as its value if it is not specified. |
 | onRender | `function` | On render callback. Keyword `this` refers to the current prompt |
-| onState | `function` | On state change callback. Function signature is an `object` with two properties: `value` and `aborted` |
+| on state | `function` | On state change callback. Function signature is an `object` with two properties: `value` and `aborted` |
```

Comment:
The program requires `onState`, so this change is wrong.

More reading on programming: https://teamtreehouse.com/community/variable-name-with-space-is-it-valid

Please take hacktoberfest as opportunity to level up your coding skills - and not as event to win a t-shirt.